### DISCUSSION TITLE
Welcome to Chime Discussions!

### DISCUSSION THREAD

**Discussion by Bluemoondragon07** [Bluemoondragon07/chime-theme#5] [Announcements]
# Welcome to Chime Discussions!
Use this page for....
- sharing your own custom snippets or variants of this theme
- non issue-related feedback or ideas
- Questions about this theme
- Anything you want to talk about, really.

To report issues in the theme, please open up a new issue [here](https://github.com/Bluemoondragon07/chime-theme/issues).

**Comment by 8bitsim** [comment]
Hi! Are the style settings not available on iOS? I can’t seem to change anything like desktop…
![Uploading A32428CC-03A8-46FA-8667-E26CD109072C.png…]()

**Reply by Bluemoondragon07** [reply]
I *think* that Style Settings works on all platforms, but I do not own an iOS device, so I'm not entirely sure. Make sure on your iOS device that the Style Settings plug-in is installed at the latest version. If Style Settings still does not work on your device, consider making an issue in the [Style Settings repository](https://github.com/mgmeyers/obsidian-style-settings/issues).
